### ISSUE TITLE
Plugins Sprint 189

### ISSUE THREAD

**Issue by ericwill** [eclipse-che/che#17678]
## Sprint goals
1. Bugs (upstream and CRW)
2. Maintenance / Automation / Technical Debt

### CRW
* [x] Upgrade Go language support plugin [CRW-1168](https://issues.redhat.com/browse/CRW-1168) @svor 
    * [x] Upgrade VS Code extension Go language support #17665
* [x] 'deploy to OpenShift' task for spring-boot-http-booster sample doesn't work [CRW-452](https://issues.redhat.com/browse/CRW-452) @svor 
* [ ] Don't open Welcome page if already open (was: Too many Welcome pages) [CRW-1029](https://issues.redhat.com/browse/CRW-1029) @vitaliy-guliy 
    * [ ] new welcome page every time when I refresh my workspace #16352

### Bugs
* [ ] Align the GitHub authentication plugin with the latest vscode GitHub PR plugin #17684 @vinokurig 
* [ ] in Java, the code snippets `try_catch`, `main` and maybe more are not working #16502 @tsmaeder 
    * [ ] Snippets don't work when Plugin Host is not colocated with Back End eclipse-theia/theia#8460
* [x] Installing Java Test Runner Support for Java #14871 @tsmaeder 
* [ ] PHP autocomplete doesn't work with latest Intelephense #17493 @ericwill 
* [ ] Cannot start any workspaces after creating an ssh key pair with a colon in the name #15665 @vitaliy-guliy 
* [x] Getting Started NodeJS+Yarn fails to run #17453 @svor 
    * [ ] Workspace Resource Monitor Plugin #17205
* [x] Che offline using an untrusted TLS cert doesn't load zip samples from internal devfile registry #16473 @vitaliy-guliy 

### Open PRs From Last Sprint
* [x] Implement Timeline API eclipse-theia/theia#7577 @vinokurig 
* [x] [Plugin-Api] Apply authentication API eclipse-theia/theia#7661 @vinokurig 

#### Theia API
* [ ] [preferences] Add ability to set default formatter eclipse-theia/theia#8354 @vitaliy-guliy 
* [ ] Add environmentVariableCollection to PluginContext eclipse-theia/theia#8405 @vinokurig 

#### Automation / Tests
* [ ] Set up PR Check for Java Che 7 plugins #15797 @svor 
    * [x] Simple LSP test suite for Java #14409
* [ ] Add plugin-registry PR check to validate new/updated vsx #17027 @ericwill 
    * [ ] Check `vscode-extensions.json` file for entries when contributing meta.yamls
    * [x] Check vsix links to prevent 404's
    * [x] Check image locations to prevent 404's
    * [ ] Check upstream GH repo against revision to make sure revisions are correct/exist
* [ ] Automate updating of sidecar images #17641 @ericwill 
    * Experiment with a GH action che-dockerfiles that opens a PR on the che-plugin-registry

#### Maintenance
* [x] Deprecate old plugins from plugin registry and remove after 3 sprints #16446 @ericwill 
* [ ] Plugin Updates
    * [ ] vscode-git and GH PR plugin, once all dependent API is merged
    * [ ] sonarlint-vscode 1.16 -> 1.17 @ericwill 
    * [x] vim 1.14.5 -> 1.16.0 @ericwill 
    * [ ] vscode-intelephense-client 1.3.11 -> 1.5.4 @ericwill 

#### Availability
| Name  | Availability | 0.6x |
| ------------- | ------------- | ------------- |
| @vitaliy-guliy  | 13  | 8 |
| @vinokurig  | 15  | 9 |
| @ericwill | 14 | 8 |
| @tsmaeder | 11 | 6 |
| @svor | 15 | 9 |

**Comment by gattytto** [comment]
@ericwill protobuf is merged, but not aded to milestones, I'm a bit lost, how do I know the PRE version it kicks in?

**Comment by ericwill** [comment]
superseded by #17832
